app/javascript/packs/complaint-sort.js

```diff
@@ -0,0 +1,13 @@
+import { table } from 'uswds';
```

Comment:
Including this line causes 2 things to happen:

1) the `aria-sort` value actually matches what is displayed. I think there may be a bug in USWDS around loading pre-sorted data
2) 2 sort buttons get added to the column headers

1 seemed more important than 2

Reply:
Pa11y may disagree with me though...